tests/foreman/cli/test_repository.py

```diff
@@ -2844,7 +2846,7 @@ def test_positive_remove_file(self, repo):
 
         :CaseImportance: Critical
         """
-        ssh.upload_file(
+        default_sat.put(
             local_file=get_data_file(RPM_TO_UPLOAD), remote_file=f"/tmp/{RPM_TO_UPLOAD}"
```

Comment:
kwargs need to be renamed (I'll stop now .... find and replace on all these :) )

Reply:
haha already did after the first comment, but great persistence!